src/main/java/jdbchomework/dao/model/MyOwnExceprion.java

```diff
@@ -0,0 +1,25 @@
+package jdbchomework.dao.model;
+
+/**
+ * Created by GetFire on 09.03.2017.
+ */
+public class MyOwnExceprion extends RuntimeException{
+    public MyOwnExceprion() {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1186)